App.js

```diff
@@ -4,6 +4,11 @@ import { Navigation } from 'react-native-navigation';
 import configureStore from './src/config/configureStore';
 import { registerScreens } from './src/screens/screens';
 import { screen, appStyle, tabsStyle } from './src/global/Screens';
+import asyncToGenerator from 'async-to-generator'
+
+// Fix for https://github.com/Bit-Nation/BITNATION-Pangea-mobile/issues/166
+// Related with https://github.com/facebook/react-native/issues/4844
+babelHelpers.asyncToGenerator = asyncToGenerator;
```

Comment:
Would you mind to try the solution provided by "skevy" https://github.com/facebook/react-native/issues/4844? I would prefere it.